extensions/ql-vscode/src/common/vscode/telemetry.ts

```diff
@@ -212,6 +212,21 @@ export class ExtensionTelemetryListener
     this.reporter.sendTelemetryErrorEvent("error", properties, {});
   }
 
+  sendConfigInformation(config: Record<string, string>): void {
+    if (!this.reporter) {
+      return;
+    }
+
+    this.reporter.sendTelemetryEvent(
+      "config",
+      {
+        ...config,
+        cliVersion: this.cliVersionStr,
```

Comment:
Would it also be useful to have the `isCanary` setting available here to check whether a certain setting is only used by canary users?

Reply:
Makes sense to me too. We may as well include `isCanary` here in all the config information events.